src/components/FlashNews.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <section class="flash-news" v-if="articles.length">
+    <div class="flash-news__name">快訊</div>
+    <div class="flash-news__container">
+      <div :style="{ transform: `translateY(${distance}%)` }" :class="[ 'flash-news__titles', { transition: isTransition } ]" @transitionend="handleTransitionEnd" @oTransitionEnd="handleTransitionEnd" @webkitTransitionEnd="handleTransitionEnd">
+        <a v-for="news in articles" :key="news.slug" :href="`${siteUrl}/story/${news.slug}`" target="_blank">{{ news.title }}</a>
+      </div>
+      <div class="flash-news__arrows">
+        <div class="next" @click="slideToNextNews(false)"></div>
+        <div class="prev" @click="slideToPrevNews"></div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import { SITE_URL } from 'src/constants'
+
+export default {
+  name: 'FlashNews',
+  props: {
+    articles: {
+      type: Array,
+      default: [],
+      required: true
+    }
+  },
+  mounted () {
+    this.AutoSlideToNextNews()
+  },
+  data () {
+    return {
+      order: 1,
+      isTransition: false,
+      isAutoSlide: true
+    }
+  },
+  computed: {
+    distance () {
+      return this.order * -100
+    },
+    siteUrl () {
+      return SITE_URL
+    }
+  },
+  methods: {
+    slideToNextNews (isAuto) {
+      if (this.isTransition) return
```

Comment:
suggest you add braces in control structure, or you can use `this.isTransition && return`